Source/WebCore/PAL/pal/spi/cocoa/WebPrivacySPI.h

```diff
@@ -127,6 +128,12 @@ typedef void (^WPStorageAccessUserAgentStringQuirksDataCompletionHandler)(WPStor
 @end
 #endif
 
+#if !defined(HAS_WEB_PRIVACY_LINK_FILTERING_RULE_PATH) && HAVE(WEB_PRIVACY_FRAMEWORK)
```

Comment:
Is the `HAVE(WEB_PRIVACY_FRAMEWORK)` important here?

Reply:
Maybe not, but whenever `HAVE(WEB_PRIVACY_FRAMEWORK)` is false this doesn’t need to be compiled, because `WPLinkFilteringRule` would be declared above.